lib/Pages/home_screen.dart

```diff
@@ -1,14 +1,15 @@
 import 'package:awesome_notifications/awesome_notifications.dart';
 import 'dart:convert';
 import 'dart:io';
-import 'package:badges/badges.dart';
+import 'package:badges/badges.dart' as badgeLibrary;
```

Comment:
This has already been implemented in previous PRs waiting to be merged

Reply:
ok fine (: